migrate/repo_checks.py

```diff
@@ -75,6 +78,83 @@ def dry_run(self):
         raise NotImplementedError
 
 
+class EnsureLabels(Check):
+    """
+    All repos in the org should have certain labels.
+    """
+
+    def __init__(self, api: GhApi, org: str, repo: str):
+        super().__init__(api, org, repo)
+        # A list of labels mapped to their hex colors
+        # so that they are the same color in all the repos.
+        # Relevant API Docs: https://docs.github.com/en/rest/issues/labels#create-a-label
+        self.labels = {
+            ":hammer_and_wrench: maintenance": "169509",
```

Comment:
Do we want to use the labels.yaml file we have?

Reply:
> the labels.yaml file we have

say more!

Reply:
The OSPR bot synchronizes labels when it looks at a repo. It uses this data file:  https://github.com/openedx/openedx-webhooks-data/blob/main/labels.yaml (private repo).   Feanil made the point that we need labels sync'ed beyond PR-activity repos, which is why this code is here, but the data can be used, right?

Reply:
Oh yeah! Having both these tools running simultaneously with different lists of labels sounds... chaotic.

Reply:
@feanil and I talked about this.

We agreed that, eventually, we should have only one tool messing with repo labels, and that it should be this tool. Feanil will write up an issue to remove that functionality from the OSPR bot, and move any remaining useful labels over to this tool.

In the meantime, Feanil will just have to be careful not to add any labels here that'll conflict with the ones the OSPR bot cares about... or else, face the consequences 🔥 💥 🚒 😱

Reply:
FYI, created https://github.com/orgs/openedx/projects/33/views/1 to fix this as a follow-on task.

Reply:
direct link to follow-on task: https://github.com/openedx/openedx-webhooks/issues/218